StrongInject.Generator/Visitors/DependencyCheckerVisitor.cs

```diff
@@ -361,6 +361,12 @@ public bool IsScopeAsync
                         result.AppendLine("'");
                         //equivalent to: `result.AppendLine($"Casting instance of Type '{ type }' to '{ forwardedType }'");`
                         break;
+                    case OwnedSource { ContainerType: var containerType }:
+                        result.Append("Resolving scope tracker '");
```

Comment:
I think we should be consistent about naming - so I would say resolving Owned, simply because that's a type that the user can search for, even if it's not necessarily the best name for this up front.

Reply:
The text `Resolving Owned 'Owned<A>'` felt repetitive. My initial thought was `Resolving 'Owned<A>'` and leaving it at that. What do you think?

Reply:
Sounds good!